presc/copies/sampling.py

```diff
@@ -40,6 +41,118 @@ def dynamical_range(df):
     return range_dict
 
 
+def find_categories(df, add_NaNs=False):
```

Comment:
This is a nit, but I'd prefer simple lower-casing `add_nans`. Stylistically it's more uniform and reduces potential for confusion/bugs.

Reply:
Done.